app/src/main/kotlin/com/wire/android/ui/home/conversations/details/editguestaccess/EditGuestAccessViewModel.kt

```diff
@@ -152,7 +152,11 @@ class EditGuestAccessViewModel @Inject constructor(
                         )
                     )
 
-                    is UpdateConversationAccessRoleUseCase.Result.Success -> Unit
+                    is UpdateConversationAccessRoleUseCase.Result.Success -> {
+                        if (!shouldEnableGuestAccess) {
+                            revokeGuestRoomLink(conversationId)
```

Comment:
i would expect the server will revoke the links automatically when disabled and all what we need to do is delete locally ?

Reply:
I don't know, you sure ? I can just clear the link locally in that case

Reply:
Let's check what other clients do, but the link being revoked automatically just sounds the right thing to me